### ISSUE TITLE
CSS breaks display of some CP entry pages

### ISSUE THREAD

**Issue by KatieMFritz** [KuriousAgency/commerce-admin-orders#12]
I haven't tested this throughout the site, but on at least some of my CP entry edit pages with Redactor fields, the responsive width is broken by this line of Modal.css:

```css
#main-container #main #header,
#main-container #main #main-content {
	width: auto;
}
```

Could you scope it to only apply to screens you're creating?

**Comment by samuelbirch** [comment]
Hi @KatieMFritz,

we'll take a look at that.

**Comment by samuelbirch** [comment]
fixed in 2.5.2
